src/site/tests/vamc/smoke.cypress.spec.js

```diff
@@ -0,0 +1,18 @@
+describe('VAMC', () => {
+  before(function() {
+    if (Cypress.env('CIRCLECI')) this.skip();
+  });
+
+  it('has expected elements', () => {
+    cy.visit('/pittsburgh-health-care/');
+    cy.viewport(481, 1000);
+
+    cy.get('#sidebar-nav-trigger').should('not.exist');
+    cy.get('#sidenav-menu').should('exist');
+    cy.get('h1').contains('VA Pittsburgh health care');
+    cy.get('h2').contains('Locations');
+    cy.get('h3').contains('Manage your health online');
+    cy.get('#stories').contains('Stories');
+    cy.get('#events').contains('Events');
+  });
+});
```

Comment:
Can we add a test that verifies the `side_bar` on mobile layout

<img width="672" alt="Screen Shot 2020-09-09 at 6 43 53 PM" src="https://user-images.githubusercontent.com/100468/92665647-ac86ff80-f2cc-11ea-9474-cae20193cea2.png">

Reply:
Please explain. I added an ID to that element and my test is checking that it exists:

```js
cy.get('#sidenav-menu').should('exist');
```

What other verification did you have in mind?

Reply:
I actually did not see the viewPort called `cy.viewport(481, 1000);` now makes sense